### ISSUE TITLE
Dilato: Core: The default version of ESLint is inconsistent with the version in the More info link

### ISSUE THREAD

**Issue by ZuoBaogang** [adobe/brackets#13657]
Steps:
1. Launch Brackets
2. Open extension manager
3. Find the ESLint under Default tab then observe the version is 3.0.5
4. Click "More info..."
5. The version is 3.1.0

Result: The default version of ESLint is inconsistent with the version in the More info link

Expect: They should keep consistent

![defaultversion](https://user-images.githubusercontent.com/17721679/30026595-30c99752-91b0-11e7-8e96-635ccd0527de.png)
![moreinfoversion](https://user-images.githubusercontent.com/17721679/30026598-31f8f974-91b0-11e7-8cc3-2e97be3cfd8f.png)

**Comment by ZuoBaogang** [comment]
ESLint appears both under Available tab and Default tab, is this by design?

**Comment by swmitra** [comment]
@ZuoBaogang Yes this is by design. We have added ESLint extension as part of default Brackets deliverable and eventually the same extension is available as user extension as part of Brackets extension registry. 
There might be users who would be using this extension before 1.11 installation. It will appear in both the tabs in those cases. 
Regarding the actual issue reported, this can't be fixed reliably. The git repository link for the extension is same in both the cases and `more Info` is a link to the GitHub page for the extension source code. When we ship Brackets, the extension version included might differ at a later point of time if the extension code is updated for any reason. That's why if ESLint extension is present as default as well as user extension, Brackets uses the user extension( more recently updated version in most of the cases).
